src/FileSystem.java

```diff
@@ -4,43 +4,81 @@
 
 public class FileSystem {
 
+    /**
+     * A function to read the data out of an existing AddressBook saved on disk.
+     * Note: this function deletes the data in the current AddressBook.
+     * 
+     * @param addressBook The AddressBook to read the data into.
+     * @param file        The file to read the data from.
+     * @throws SQLException          Thrown if the data in the file is not in the
+     *                               expected SQL format
+     * @throws FileNotFoundException Thrown if the file does not exist or is
+     *                               unreadable
+     */
```

Comment:
All code is supposed to have java docs like this. I haven't got to your files yet, but we need to go through and add comments everywhere!